.jscsrc

```diff
@@ -101,6 +101,5 @@
         "that"
     ],
     "validateIndentation": 4,
-    "validateLineBreaks": "LF",
```

Comment:
I noticed the removal in the other PR #93 and went with it, to get the build done

Reply:
reverted as well